src/persist-client/src/internal/trace.rs

```diff
@@ -268,28 +262,6 @@ impl<T: Timestamp + Lattice> Trace<T> {
         }
         ret
     }
-
-    #[allow(dead_code)]
-    pub fn describe(&self) -> String {
```

Comment:
Dropped this mostly for expediency, but I'm happy to bring it back if we have plans for it!

Reply:
I found it quite useful while working on various compaction 2.0 branches, but since you'll be doing more of that now, happy to defer to you on the usefulness!